lib/internal/modules/cjs/loader.js

```diff
@@ -978,6 +989,7 @@ Module._load = function(request, parent, isMain) {
   return module.exports;
 };
 
+const cjsConditions = new Set(['require', 'node']);
```

Comment:
```suggestion
// TODO: Use this set when resolving pkg#exports conditions.
const cjsConditions = new Set(['require', 'node']);
```

Can we add a TODO to also use this set in line 611? https://github.com/nodejs/node/blob/90763bbbc4e83cdb56a3a5ac5a1f5cddaea64b8b/lib/internal/modules/cjs/loader.js#L611-L630